var/spack/repos/builtin/packages/binutils/package.py

```diff
@@ -46,6 +46,7 @@ class Binutils(Package):
     # OpenSpeedShop and cbtf-krell
     variant('krellpatch', default=False,
             description="build with openspeedshop based patch.")
+    variant('plugins', default=False)
```

Comment:
Sorry I don't have one, it is needed to build the gold linker c.f. https://github.com/unsw-corg/SVF/wiki/Install-LLVM-Gold-Plugin-on-Ubuntu